### PR TITLE
Added orders to assets

### PR DESCRIPTION
**Pull request by Trokkin**
For easy usage of item orders and impossibility to mistake string orders (which also looks worse for me).

### CODE REVIEW

File: wurst/_wurst/Assets.wurst [hunk 1]
```diff
@@ -16222,3 +16222,380 @@ public class Icons
 	static constant pASBTNVampiricAura = "ReplaceableTextures\\PassiveButtons\\PASBTNVampiricAura.blp"
 	static constant pASBTNVorpalBlades = "ReplaceableTextures\\PassiveButtons\\PASBTNVorpalBlades.blp"
 	static constant pASBTNWarStomp = "ReplaceableTextures\\PassiveButtons\\PASBTNWarStomp.blp"
+
+
+/*
+	Source: cj_order.j v 0.12
```
Comment (Frotty) [inline wurst/_wurst/Assets.wurst]: I don't think this belongs into "assets", since they aren't actually resources.
I think this can be put into a separate package.

File: wurst/_wurst/Assets.wurst [hunk 2]
```diff
@@ -16222,3 +16222,380 @@ public class Icons
...
+	2009 Shadow Daemon \\ cjass.xgm.ru
+*/
+
+/** Class that contains every known order in game. */
+public class Orders
+	static constant smart					= 851971
+	static constant stop					= 851972
+	static constant setrally				= 851980
+	static constant getitem					= 851981
+	static constant attack					= 851983
```
Comment (Frotty) [inline wurst/_wurst/Assets.wurst]: please fix the mixed indentations here

File: wurst/_wurst/Assets.wurst [hunk 3]
```diff
@@ -16222,3 +16222,380 @@ public class Icons
...
+	static constant creepheal				= 852248
+	static constant creephealon				= 852249
+	static constant creephealoff			= 852250
+	static constant creepthunderbolt		= 852252
+	static constant creepthunderclap		= 852253
+	static constant poisonarrowstarg		= 852254
+	static constant poisonarrows			= 852255
+	static constant unpoisonarrows			= 852256
+	/** 'AIaa' (Book of attack) ability order. Source: https://www.hiveworkshop.com/threads/snippet-add-base-attack.289416/ */
+	static constant aIaa					= 852259
```
Comment (Frotty) [inline wurst/_wurst/Assets.wurst]: give them a proper name

File: wurst/util/Printing.wurst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 package Printing
 import NoWurst
 
-@configurable public var DEBUG_LEVEL = Loglevel.INFO
```
Comment (Frotty) [inline wurst/util/Printing.wurst]: Why did you change this? It's configurable. if you want to change it you can configure it inside your project. Default should stay INFO.

Comment (Trokkin) [reply wurst/util/Printing.wurst]: Oh. I shouldn't commit it at all, sorry.

File: wurst/_wurst/Assets.wurst (hunk 2 above)
Comment (Trokkin) [reply wurst/_wurst/Assets.wurst]: it's strange because in my IDE its all right with indentations.

File: wurst/_wurst/Assets.wurst (hunk 1 above)
Comment (Trokkin) [reply wurst/_wurst/Assets.wurst]: Hm... Where should it be placed then?

File: wurst/dummy/InstantDummyCaster.wurst [hunk 5]
```diff
@@ -16,51 +16,55 @@ public class InstantDummyCaster
```
Comment (Frotty) [inline wurst/dummy/InstantDummyCaster.wurst]: Belongs into it's own PR?

File: wurst/_wurst/Orders.wurst [hunk 6]
```diff
@@ -0,0 +1,379 @@
...
+import NoWurst
+
+/** Class that contains every known order in game that has no string counterpart */
+public class SpecialOrders
+	static constant tomeOfAttack			 = 852259
+	static constant wandOfIllusion			 = 852274
+
+
+
+/*
```
Comment (Frotty) [inline wurst/_wurst/Orders.wurst]: please remove this and just add 1 line of credit to the guy after imports like other packages

File: wurst/_wurst/Orders.wurst [hunk 7]
```diff
@@ -0,0 +1,379 @@
...
+	Source: cj_order.j v 0.12
+		( Orders Define Library )
+
+    This was taken from cJass standard library
+	2009 Shadow Daemon \\ cjass.xgm.ru
+*/
+/** Class that contains every known order in game. */
+public class Orders extends SpecialOrders
```
Comment (Frotty) [inline wurst/_wurst/Orders.wurst]: the indentation is still mixed: https://i.imgur.com/eNy9ZIG.png
See yourself https://github.com/wurstscript/WurstStdlib2/pull/21/files

File: wurst/dummy/InstantDummyCaster.wurst (hunk 5 above)
Comment (Trokkin) [reply wurst/dummy/InstantDummyCaster.wurst]: It should be, probably... Though this PR is almost meaningless without it.

File: wurst/dummy/InstantDummyCaster.wurst [hunk 8]
```diff
@@ -16,51 +16,55 @@ public class InstantDummyCaster
...
+		id = abilityId
 		caster.addAbility(id)
+		caster.setXY(pos)
 		if lvl > 1
 			caster.setAbilityLevel(id, lvl)
 		if owner != null
 			caster.setOwner(owner, false)
 			
 	private static function finish()
-		caster..setOwner(DUMMY_PLAYER, false)..removeAbility(id)
```
Comment (Frotty) [inline wurst/dummy/InstantDummyCaster.wurst]: Why did you convert ".." to "caster." ????
.. is the recommended convention, please restore it.

File: wurst/_wurst/Orders.wurst [hunk 9]
```diff
@@ -0,0 +1,371 @@
...
+	static constant selfdestruct			 = 852040
+	static constant selfdestructoff			 = 852042
+	static constant selfdestructon			 = 852041
+	static constant sentinel			 = 852182
+	static constant setrally			 = 851980
+	static constant shadowsight			 = 852570
+	static constant shadowstrike			 = 852527
+	static constant shockwave			 = 852125
+	static constant silence				 = 852592
+	static constant sleep				 = 852227
```
Comment (Frotty) [inline wurst/_wurst/Orders.wurst]: where is `stun` ?

File: wurst/_wurst/Orders.wurst [hunk 10]
```diff
@@ -0,0 +1,371 @@
+package Orders
+// Credits Shadow Daemon for cj_order.j from cJass
+import NoWurst
+
+/** Class that contains every known order in game that has no string counterpart */
+public class SpecialOrders
```
Comment (Frotty) [inline wurst/_wurst/Orders.wurst]: Is there anything else special about them? 
Maybe you could make another class containing UI order (move, attack, smart)

File: wurst/_wurst/Orders.wurst (hunk 9 above)
Comment (Trokkin) [reply wurst/_wurst/Orders.wurst]: whoa, I didn't know about it. Anyway, I've taken the order list from cjass, and it can miss much of them, so I assume it can be added later, or if I will be given an id of that order.

Comment (Frotty) [reply wurst/_wurst/Orders.wurst]:
```
Hidden Order Ids:
851971 (smart): This is a point or object targeted order that is like a right click.
852000 (skillmenu): This is an order with no target that opens the skill menu of heroes. If it is issued for a normal unit with triggers it will black out the command card for this unit, the command card will revert to normal after reselecting the unit.
851994 (buildmenu): This is an order with no target that opens up the build menu of a unit that can build structures.
852002 to 852007 (moveslot): These are item targeted orders that move the target item to a certain inventory slot of the ordered hero. The id 852002 will move it to slot 1, the id 852003 will move it to slot 2 and so on.
852008 to 852013 (useslot): These are orders that will make the ordered hero use the item in a certain inventory slot. If it's an order with no target or object or point targeted depends on the type of item. The id 852008 will use the item in slot 1, the id 852009 will use the item in slot 2 and so on.
851976 (cancel): This is an order with no target that is like a click on a cancel button. We used to be able to catch cancel clicks with this id back then but this id doesn't seem to work any more.
851973 (stunned): This order is issued to units that get stunned by a spell, for example War Stomp (AOws). This is probably a hold position + hold fire order. The ordered unit will be unable to move and attack.
```

From: http://www.wc3c.net/showthread.php?t=81742

File: wurst/_wurst/Orders.wurst (hunk 10 above)
Comment (Trokkin) [reply wurst/_wurst/Orders.wurst]: There's nothing more than it really. It was the first name that came to my mind and I don't really care about it since it is included into Orders class, so feel free to suggest something else.
Well, it intends strict hierarchy if I want Orders class to extend every other class with orders, so I thought: if it is not usual orders, then it should be some *special* orders.

**Comment by Trokkin** [comment]
Is there something else what is to be done with this PR?

**Comment by Frotty** [comment]
A short explanation of `itemdrag` and `itemuse` plus example on how to use would be nice. But it's already looking great now and I can merge if you want 👍 :)

**Comment by Trokkin** [comment]
Yay :D
Added last part of comments. The rest of them are on you now ;)

**Comment by Frotty** [comment]
Don't close PRs, wait for them to be merged ;)
